### PR TITLE
[14.0][FIX] contract_sale: Fixed the record contract_modification at .csv file

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
the "contract" module, had added a field called contract_modification, and the "contract_sale" module was not updated to that change.
@FORGEFLOW
@AaronHForgeFlow 
@BernatPForgeFlow 
@mariadforgeflow
@LoisRForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by BernatPForgeFlow — APPROVED** [review]

File: contract_sale/security/ir.model.access.csv [hunk 1]
```diff
@@ -2,6 +2,7 @@
 "contract_template_salesman","Recurring Salesman","contract.model_contract_template","sales_team.group_sale_salesman",1,1,1,0
 "contract_template_sale_manager","Recurring Sale Manager","contract.model_contract_template","sales_team.group_sale_manager",1,1,1,1
 "contract_line_saleman","Recurring Invoice Line Saleman","contract.model_contract_line","sales_team.group_sale_salesman",1,1,1,0
+"contract_model_modification","Contract Model Modification","contract.model_contract_modification","sales_team.group_sale_salesman",1,1,1,0
```
Comment (AaronHForgeFlow) [inline contract_sale/security/ir.model.access.csv]:
```suggestion
"contract_modification_salesman","Contract Modification Salesman","contract.model_contract_modification","sales_team.group_sale_salesman",1,1,1,0
```

Sorry for the nitpicking, the rule is fine bu in the id and the name I think it is best to put the model name and the name of the group.

Comment (JoanMForgeFlow) [reply contract_sale/security/ir.model.access.csv]: Thanks! done :)

**Review by AaronHForgeFlow — APPROVED** [review]
/ocabot merge patch

**Review by JoanMForgeFlow — COMMENTED** [review]
@pedrobaeza could we merge this?

**Comment by pedrobaeza** [comment]
@AaronHForgeFlow the bot commands don't work on the approval comment.

/ocabot merge minor

**Comment by JoanMForgeFlow** [comment]
> @AaronHForgeFlow the bot commands don't work on the approval comment.
> 
> /ocabot merge minor

thanks Pedro!

**Comment by JoanMForgeFlow** [comment]
> @AaronHForgeFlow the bot commands don't work on the approval comment.
> 
> /ocabot merge minor
Thanks pedro!

**Pull request merged**